src/content/docs/alerts/create-alert/create-alert-condition/alert-conditions.mdx

```diff
@@ -374,9 +377,179 @@ For all alert conditions built with NRQL queries, the <DNT>**Signal history**</D
+
+    By default, the selected alert policy will add the copied condition in the **Disabled** state. Follow standard procedures to [add](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/define-alert-conditions) or copy more conditions to the alert policy, and then [**Enable** the condition](/docs/alerts/new-relic-alerts-beta/updating-alert-policies/disable-or-delete-alert-policies-conditions#disable-conditions) as needed. Additionally, the new condition will not copy any [tags](/docs/new-relic-solutions/new-relic-one/core-concepts/use-tags-help-organize-find-your-data/#add-via-ui-api) added to the original condition.
+
+## Configure an existing condition [#configure-condition]
+
+To change a condition, go to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Alerts > Alert Conditions**</DNT> and click a condition.
+
+* To [change the condition's name](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/define-alert-conditions#rename-condition) select <DNT>**Edit**</DNT> in the ellipses (...) menu for the condition in the list.
+* To update the values for the required Critical (red) or optional Warning (yellow) [thresholds](/docs/alerts/new-relic-alerts-beta/configuring-alert-policies/define-thresholds-trigger-alert), select <DNT>**Edit**</DNT> in the ellipses (...) menu for the condition in the list..
+* To update the condition's [runbook URL](/docs/alerts/new-relic-alerts-beta/reviewing-alert-incidents/provide-runbook-instructions-alerts), select <DNT>**Edit**</DNT> in the ellipses (...) menu for the condition in the list.
+
+## Enable/disable a condition [#disable-conditions]
+
+You can enable <img title="Alerts v3: Condition enabled" alt="041715icon-condition-on" src={accountsConditionOn}/> or disable <img title="Alerts v3: Condition disabled" alt="041715icon-condition-off" src={accountsConditionOff}/> any conditions, and the policy will continue to apply. To disable or re-enable a condition:
+
+1. Go to <DNT>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Alerts > Alert Conditions**</DNT>. Then, from the list of <DNT>**Alert conditions**</DNT>, use the toggle to enable or disable the condition.
+2. Click the <DNT>**On/Off**</DNT> switch to toggle it.
+
+If you [copy a condition](/docs/alerts/new-relic-alerts-beta/updating-alert-policies/copy-alert-conditions), it automatically saves it in the new policy as disabled (<DNT>**Off**</DNT>), even if the condition was enabled (<DNT>**On**</DNT>) in the original policy.
+
+## Delete a condition [#delete-conditions]
+
+If a policy has multiple conditions, you can delete any number of them, and the remaining conditions for the policy will continue to apply. To turn a condition off but keep it with the policy, [disable](/docs/alerts/new-relic-alerts-beta/updating-alert-policies/disable-or-delete-alert-policies-conditions#condition-on-off) it. To delete one or more conditions:
```

Comment:
`and the remaining conditions for the policy will continue to apply.` is it necessary to say this?

Reply:
You're right, it's not necessary. I am deleting this.